pkg/sqlite/load.go

```diff
@@ -424,7 +424,7 @@ func (s *sqlLoader) AddPackageChannelsFromGraph(graph *registry.Package) error {
 	}
 
 	if err := updateDefaultChannel(tx, graph.DefaultChannel, graph.Name); err != nil {
-		errs = append(errs, err)
+		//errs = append(errs, err)
```

Comment:
Do we need to remove this commented-out code?

Reply:
It is wrong. Good catcher. 
I am retesting and updating the first comment